products/installation.yaml

```diff
@@ -80,7 +80,7 @@ products:
     channel: "rhmi"
     installFrom: "index"
     package: "rhmi-cloud-resources"
-    index: "quay.io/integreatly/cloud-resource-operator:index-v0.37.1"
+    index: "quay.io/kevfan/cloud-resource-operator:index-v0.37.1"
```

Comment:
Using own custom index for now that is built from the CRO STS feature branch - https://github.com/integr8ly/cloud-resource-operator/pull/315 

This should be updated when the CRO STS feature branch is merged and a CRO release is created